public/stylesheets/stops.css

```diff
@@ -0,0 +1,63 @@
+#stops-sidebar, #stops-sidebar > div {
```

Comment:
We'll probably want to merge `stops.css` and `routes.css` in the future since they contain a lot of duplicate CSS, but we can do that in another PR--no need to clutter up this one.